server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -325,14 +324,13 @@ static TransportVersion def(int id) {
     public static final TransportVersion ESQL_PROFILE_INCLUDE_PLAN = def(9_111_0_00);
     public static final TransportVersion MAPPINGS_IN_DATA_STREAMS = def(9_112_0_00);
     public static final TransportVersion ESQL_SPLIT_ON_BIG_VALUES_9_1 = def(9_112_0_01);
-    public static final TransportVersion ESQL_FIXED_INDEX_LIKE_9_1 = def(9_112_0_02);
+    // If you need to backport to 9.1 the next patch version is 9_112_0_08
```

Comment:
I have mixed feelings on this. Mainly because there's nothing guarnateeing it's up to date. For example, when we release 9.1.4 and bump the versions this will change. I think we're better off just having folks go look at the 9.1 branch to determine this.

Reply:
That's fair. I'll remove it.